### ISSUE TITLE
Why the saved model in "data" folder is not .ckpt, but ckpt.meta?

### ISSUE THREAD

**Issue by brisker** [sugyan/tensorflow-mnist#6]
I am  new to tensorflow and I see that in the "data" folder it is like this after I delete the pretrained ckpt files and train it by myself:
![image](https://cloud.githubusercontent.com/assets/13804492/23245945/50147e60-f9cb-11e6-9128-59accd1d0f75.png)
So how to generate these pure .ckpt files?
![image](https://cloud.githubusercontent.com/assets/13804492/23245973/6775f700-f9cb-11e6-8a8f-440e3b8b5d05.png)
Thanks!

**Comment by sugyan** [comment]
Oops, 
The checkpoint format of `tf.train.Saver` changed from TensorFlow 0.12. Since data folder in this repository is a little old, it contains the data outputted in format of Saver V1. I'll fix it later, sorry.
But don't worry, both formats are able to be read.
see: https://github.com/tensorflow/tensorflow/blob/v1.0.0/RELEASE.md#release-0120

**Comment by brisker** [comment]
@sugyan 
Just modify the name "convolutional.ckpt.data-00000-of-00001" to "convolutional.ckpt" ?
Besides,
 All I want to do is train a simple mnist model save it as a ckpt file, then load the model and a new image,  predict the 28*28 image , and print its corresponding number. I tried this, but why so hard?.....

any help?
Or could you please give a demo how to achieve this? The code I use is like this:

[inference.txt](https://github.com/sugyan/tensorflow-mnist/files/795821/inference.txt)
I get error like this:
DataLossError (see above for traceback): Unable to open table file /home/jcc/tensorflow/TensorFlow-Examples/examples/tensorflow-mnist-master/mnist/data/convolutional.ckpt: Data loss: not an sstable (bad magic number): perhaps your file is in a different file format and you need to use a different restore operator?
	 [[Node: save_7/RestoreV2_19 = RestoreV2[dtypes=[DT_FLOAT], _device="/job:localhost/replica:0/task:0/cpu:0"](_recv_save_7/Const_0, save_7/RestoreV2_19/tensor_names, save_7/RestoreV2_19/shape_and_slices)]]
	 [[Node: save_7/RestoreV2_23/_81 = _Recv[client_terminated=false, recv_device="/job:localhost/replica:0/task:0/gpu:0", send_device="/job:localhost/replica:0/task:0/cpu:0", send_device_incarnation=1, tensor_name="edge_275_save_7/RestoreV2_23", tensor_type=DT_FLOAT, _device="/job:localhost/replica:0/task:0/gpu:0"]()]]

**Comment by sugyan** [comment]
@brisker No, you don't need to rename files. When "model.ckpt" is specified as the argument of `saver.restore`, TensorFlow >= 0.12 searches for and reads the file "model.ckpt.data-00000-of-00001".

**Comment by sugyan** [comment]
checkpoint data is updated. https://github.com/sugyan/tensorflow-mnist/commit/2bbcacab79a53ea939cbee96c6c8d17f85e686bf
